react/components/NotificationBar/index.js

```diff
@@ -0,0 +1,71 @@
+import React, { memo } from 'react'
+import { bool, string, oneOf } from 'prop-types'
+import classNames from 'classnames'
+import { values } from 'ramda'
+
+import styles from './styles.css'
+
+const getEnumValues = enumObject => values(enumObject).map(({ value }) => value)
+const getEnumNames = enumObject => values(enumObject).map(({ name }) => name)
+
+const colorPatterns = {
+  ACTION_PRIMARY: {
```

Comment:
Primary looks odd to me, isn't better use the secondary color? Either way I'm not a designer, are them ok with it?

Reply:
I'm keeping the same name used by the semantic colors of tachyons.

Reply:
I'm talking about the color itself not the name

Reply:
The correct is allow the user to choose the color, but for this first version, I used the colors that allow more contrast in the page.
This bar is like a banner, so you want to give a spotlight, and with primary-color and base-inverted you achieve this.

Reply:
@gustavorosolem so, I think you should leave it only at `bg-base--inverted` for now, and allow for arbitrary colors and/or more tokens later. 

If you add support for `action-primary` now, removing it later will be a breaking change, and I think we should think it through before adding support for it

Reply:
so this also means, remove the option for different colors for now